Components/Queues/Drivers/PhpAmqpLibDriver.php

```diff
@@ -0,0 +1,296 @@
+     * The server will send a message in advance if it is equal to or smaller in size than the available prefetch size
+     * (and also falls into other prefetch limits).
+     * May be set to zero, meaning "no specific limit", although other prefetch limits may still apply.
+     * The prefetch-size is ignored if the no-ack option is set.
+     */
+    const PREFETCH_SIZE = null;
+
+    /**
+     * Represents the amount of message to consume by iteration
+     */
+    const PREFETCH_COUNT = 10;
+
+    /*
+     * Default exchange name
+     */
+    const EXCHANGE_NAME = '';
+
+    /**
+     * Default port to connection
+     */
+    const DEFAULT_PORT = 5672;
+
+    /**
+     * Default host to connect
+     */
+    const DEFAULT_HOST = 'localhost';
+
+    /**
+     * @var string
+     */
+    protected $format = AsyncQueueDriverInterface::FORMAT_JSON;
+
+    /**
+     * @var AMQPChannel|null
+     */
+    private $channel;
+
+    /**
+     * @var array|null
+     */
+    protected $connectionsData;
+
+    /**
+     * @var AMQPStreamConnection
+     */
+    protected $stream;
+
+    /**
+     * @throws \Exception
+     */
+    public function __destruct()
+    {
+        $this->disconnect();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function configure(string $host, string $username, string $password, string $vhost = null)
+    {
+        $urls = explode(',', $host);
+
+        foreach ($urls as $url) {
+            $parsedUrl = parse_url($url);
+
+            $this->connectionsData[] = [
+                'host' => $parsedUrl['host'] ?? self::DEFAULT_HOST,
+                'user' => $username,
+                'password' => $password,
+                'vhost' => $vhost,
+                'port' => $parsedUrl['port'] ?? self::DEFAULT_PORT,
+            ];
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     * @throws \Exception
+     * @throws AMQPProtocolException
+     */
+    public function connect($shuffle = true)
+    {
+        $this->shuffleConnections($shuffle);
+        try {
+            $this->stream = AMQPStreamConnection::create_connection($this->connectionsData, []);
+        } catch (AMQPIOException $exception) {
+            throw new AMQPProtocolException($exception->getCode(), $exception->getMessage(), null);
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     * @throws \Exception
+     * @throws \Exception
+     */
+    public function disconnect()
+    {
+        if ($this->validateConnection()) {
+            $this->stream->close();
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function createQueueMessage(): QueueMessageInterface
+    {
+        $msg = new QueueMessage();
+        $msg->setContext(new Context());
+
+        return $msg;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isConnected(): bool
+    {
+        return $this->validateConnection() && $this->stream->isConnected();
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
I see in the code you write in 2 different ways the inherit doc comment
- {@inheritDoc}
- {@inheritdoc}
Basically the D in 1 case is capital letter and in another case lower case

Do you know if both are supported? In phpdoc I see that they mention {@inheritdoc} https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html

Reply:
Ok, fixed.